contents/docs/product-analytics/troubleshooting.mdx

```diff
@@ -63,49 +63,51 @@ posthog = PostHog::Client.new({
 
 </MultiLanguage>
 
-3. Make sure that the SDK sends a couple of events.
-4. See if the incoming events arrive as expected.
+3. Send a few events from your app.
+
+If no events are *not* visible in webhook.site, that means you have misconfigured your PostHog library. Consult our [installation guide](/docs/product-analytics/installation) for how to initialize PostHog.
+
+If events are indeed visible in webhook.site, these means that you have configured your PostHog correctly.
 
 ![Events visible on webhook.site](../../images/docs/product-analytics/events-in-webhook.site.png)
 
-### 2. There is a bug within the library being used.
+### 2. There is a bug the library
+
+Sometimes, bugs in the PostHog library may be the cause of missing events. However, this is quite uncommon.
 
-Certain events could fail to reach PostHog if there is a bug in the used library. (this is quite uncommon)
+To debug if this is the issue, most of our SDKs have an option to enable debug logging. Consult the relevant [SDK docs](/docs/libraries) for how to do this.
 
-#### Web libraries
-1. Open the devtools and switch to the network tab.
-2. Look for the failed requests to app.posthog.com/e (or eu.posthog.com/e if using the EU Cloud).
+If you're using the [JavaScript web](/docs/libraries/js) library, you can identify failed requests using the network tab in your browser devtools:
+
+1. Open devtools in your browser and switch to the network tab.
+2. Identify the failed requests to app.posthog.com/e (or eu.posthog.com/e if you're using EU Cloud).
 3. Right click one of those request and click Copy -> Copy as cURL.
+4. See below on how to [report your issue](/docs/product-analytics/troubleshooting#report-your-specific-issue) and attach your cURL command to get further support.
 
 ![network request being sent to posthog](../../images/docs/product-analytics/network-request-to-posthog.png)
 
-3. See how to [Report your specific issue](/docs/product-analytics/troubleshooting#report-your-specific-issue) to get further support.
+### 3. Events are not ingested after being received by PostHog.
 
-#### Other libraries
-See the respective debugging guides, e.g. [Node.js debugging](https://posthog.com/docs/libraries/node#debugging-and-exceptions).
+If your library is configured correctly and successfully sending events to PostHog, events may get lost in the ingestion pipeline – e.g. if there is an [active incident](https://status.posthog.com).
```

Comment:
I think it's event more important to call the library bug problem out for non-web libraries as we're more likely to have bugs there 😅 

Note that a typical backend library problem is that events weren't flushed, which is (1), but they will likely think it's (2).